pagseguro/__init__.py

```diff
@@ -32,8 +32,12 @@ def parse_xml(self, xml):
             return
 
         transaction = parsed.get('transaction', {})
-        for k, v in transaction.iteritems():
-            setattr(self, k, v)
+        try:
```

Comment:
Aqui podemos deixar só o **items()** mesmo que funcionará na versão 2 e 3 do Python.  Você pode mudar para apenas: `for k, v in transaction.items()` sem a necessidade do try e comitar novamente? valeu!